noir-projects/aztec-nr/uint-note/src/uint_note.nr

```diff
@@ -201,12 +204,16 @@ pub struct PartialUintNote {
 }
 
 impl PartialUintNote {
-    pub fn commitment(self) -> Field {
-        self.commitment
+    /// Computes a partial note validity commitment. This commitment is stored in public storage when creating the
+    /// partial note in private and then is used to check that completer and note are valid upon partial note
+    /// completion in public (completer is the entity that can complete the partial note).
+    pub fn compute_validity_commitment(self, completer: AztecAddress) -> Field {
```

Comment:
Introduced a `completer` term since I needed to differentiate from `from` for it all to not be super twisted.